lib/features/upload/domain/usecases/local_file_picker_interactor.dart

```diff
@@ -13,23 +14,21 @@ class LocalFilePickerInteractor {
 
   Stream<Either<Failure, Success>> execute({FileType fileType = FileType.any}) async* {
     try {
+      yield Right<Failure, Success>(LocalFilePickerLoading());
+
       final filesResult = await FilePicker.platform.pickFiles(
         type: fileType,
         allowMultiple: true,
         withData: PlatformInfo.isWeb
       );
```

Comment:
```suggestion
      final filesResult = await FilePicker.platform.pickFiles(
        type: fileType,
        allowMultiple: true,
        withData: false,
        withReadStream: true,
      );
```

Reply:
and support `final Stream<List<int>>? readStream;` in FileInfo

Reply:
done